drivers/syslog/vsyslog.c

```diff
@@ -67,6 +67,8 @@ static FAR const char * g_priority_str[] =
 int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 {
   struct lib_syslogstream_s stream;
+  char buffer[128];
```

Comment:
Please use some macro similar to SYSLOG_RPMSG_MAXLEN instead of hard-coding it here to 128

Reply:
I noticed there is a CONFIG_SYSLOG_INTBUFSIZE for interrupt buffer, could it be used? Is there some way to avoid allocating this new buffer is using some other pre-allocated buffer?

Reply:
This buffer only stores some syslog prefixes (e.g. timerstamp pid priority), its maximum length is fixed and less than 128

The parameters actually passed in by syslog are printed by lib_vsprintf

Reply:
let's use a macro

Reply:
done